polaris-tokens/src/themes/base/zIndex.ts

```diff
@@ -23,6 +24,9 @@ export type ZIndexTokenGroup = {
 export const zIndex: {
   [TokenName in ZIndexTokenName]: MetaTokenProperties;
 } = {
+  'z-index-0': {
+    value: 'auto',
```

Comment:
Instead of setting `0` here, I've opted for `auto` [which sets the value to 0 but doesn't create a new stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index#values). This is to match the behaviour of `border-radius` which sets `none` instead of `0`, and follows the principle of _"Do what the user means, not what the user said"_.

Am happy to be challenged on this for both questions of:

1. Do we even need a `0` zindex?
2. Should this _and_ `border-radius` just be `0` values?